### PR TITLE
Live update #1 2023

### PR DESCRIPTION
**Pull request by kpodemski**
Announcement of the first Live Update of the year.

### CODE REVIEW

**Review by matks — APPROVED** [review]

File: content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "Announcing January PrestaShop Project Live Update"
+subtitle: "Let's meet for the first Live Update in 2023"
+date:   2023-01-06 09:00:00
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community]
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]:
```suggestion
tags: [contribute, event, community, live-update]
```

Comment (eternoendless) [reply content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]: Remember to add the tags here: https://github.com/PrestaShop/prestashop.github.io/blob/master/data/tags.yml

File: content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+Dear community,
+
+The first Live Update for the PrestaShop Project will happen on [**Wednesday, January 11th, 2023 at 4pm CET**](https://www.youtube.com/watch?v=YDS51o48W1I)!
+It was decided to switch from _Public Demo_ to _Live update_, as it would be great to experiment with formats other than you could see in previous sessions.
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]: Why not just say that the public demo is now called live update?

File: content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+subtitle: "Let's meet for the first Live Update in 2023"
+date:   2023-01-06 09:00:00
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+Dear community,
+
+The first Live Update for the PrestaShop Project will happen on [**Wednesday, January 11th, 2023 at 4pm CET**](https://www.youtube.com/watch?v=YDS51o48W1I)!
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]: For timezones, consider using time.is: https://time.is/1600_11_Jan_2023_in_Paris

Comment (kpodemski) [reply content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]: good idea, done

File: content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md (hunk 2 above)
Comment (kpodemski) [reply content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md]: I think it's worth explaining :) I've improved this part.

**Pull request merged**